tests/Microsoft.Identity.Test.Integration.net45/HeadlessTests/ConfidentialClientIntegrationTests.cs

```diff
@@ -162,14 +181,16 @@ public async Task ConfidentialClientWithRSACertificateTestAsync()
         }
 
         [TestMethod]
-        public async Task ConfidentialClientWithClientSecretTestAsync()
+        [DataRow(ConfidentialClientID, TestAuthority, "default", DisplayName ="Public Cloud")]
```

Comment:
Can you rename the existing constants as "PublicClould" to be more consistent with this model, e.g. `PublicCloudAuthority` etc.